spooky-doodle/Bangazon-Platform-API
# Implement customers

# Description

Implemented all features desired on `api/customers` endpoint.

Fixes #2 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1.  Run testing_bangazon.sql to clean the database.
2.  Run all TestCustomers unit tests.  All should pass.



# Checklist:
